x-pack/test/case_api_integration/common/lib/utils.ts

```diff
@@ -600,6 +600,27 @@ export const createCase = async (
   return theCase;
 };
 
+/**
+ * Sends a delete request for the specified case IDs.
+ */
+export const sendDeleteCasesRequest = async ({
```

Comment:
We already have a function called `deleteCase` so I went with this name, happy to change it though.